vividus-plugin-rest-api/src/main/java/org/vividus/http/validation/ResourceValidator.java

```diff
@@ -104,4 +104,9 @@ private HttpResponse executeHttpMethod(HttpMethod httpMethod, HttpClientContext
         ClassicHttpRequest request = httpMethod.createRequest(uri);
         return httpClient.execute(request, httpClientContext);
     }
+
+    public void clearCache()
```

Comment:
why is it needed?

Reply:
[Validator](https://github.com/vividus-framework/vividus/blob/99df3bbcf5a008c58969af4d12376398b8e27edb/vividus-plugin-rest-api/src/main/java/org/vividus/http/validation/ResourceValidator.java#L52) cache can store or [WebPageResourceValidation](https://github.com/vividus-framework/vividus/blob/ada61576f8f8ddec1c951dd50bcfd1d3c7a45a4e/vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/validator/model/WebPageResourceValidation.java) or [ResourceValidation](https://github.com/vividus-framework/vividus/blob/ada61576f8f8ddec1c951dd50bcfd1d3c7a45a4e/vividus-plugin-rest-api/src/main/java/org/vividus/http/validation/model/ResourceValidation.java) (for different validation steps) and if we're using different steps for validation cached result from one step may be used in another step with class cast exception.